html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML Forms
+    </title>
+  </head>
+  <body>
+    <h1>
+      Coding Form!
+    </h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input type="text" name="email" />
+        </div>
+      </div>
+      <div>
+        <label>Why is coding cool?</label>
+        <div>
+          <select name="reasons">
+            <option value="choose">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </div>
+      </div>
+      <div>
+        <label>Favorite Language (choose one)</label>
+        <div>
+          <input type="radio" name="language" value="HTML" /> HTML
+          <input type="radio" name="language" value="CSS" /> CSS
+          <input type="radio" name="language" value="JS" /> JavaScript
+        </div>
+      </div>
+      <div>
+        <label>Would you like to sign up for more information?</label>
+        <div>
+          <input type="checkbox" name="signup" value="yes" /> Sign me up!
+        </div>
+      </div>
+      <div>
+        <label>Comments</label>
+        <div>
+          <textarea name="comments" cols="25" rows="10"></textarea>
+        </div>
+      </div>
+      <div>
+        <button>Submit</button>
```

Comment:
It's generally a good idea to specify the `type` attributes on `<button>`s, even if they have the default `type`.